### ISSUE TITLE
[Accessibility][PeoplePicker] noResultsFoundText is not always announced

### ISSUE THREAD

**Issue by somoo88** [microsoft/fluentui#17780]
### Environment Information

- **Package version(s)**: "@fluentui/react": "^7.129.0",
- **Browser and OS versions**: Edge v89.0.774.57

### Describe the issue:

Using noResultsFoundText in the people picker suggestion props does not announce properly when no results are shown.

### Please provide a reproduction of the issue in a codepen:

https://codepen.io/ayeandy/pen/poRaXey

#### Actual behavior:

Screen reader announces the "no suggestions, this should be announced always" first time, but not subsequent times.

#### Expected behavior:

Screen reader should always announce the "no suggestions, this should be announced always."

### Documentation describing expected behavior

Our accessibility test pass has revealed that no suggested items are not always announced. While I understand for normal results that the screen reader is not being announced as to not overwhelm users with picker results updates, no results should be considered to announce always.
